app/src/main/java/com/fcaputo/parkingapp/mvp/model/ReservationModel.kt

```diff
@@ -0,0 +1,58 @@
+package com.fcaputo.parkingapp.mvp.model
+
+import com.fcaputo.parkingapp.entity.Reservation
+import com.fcaputo.parkingapp.mvp.contract.ReservationContract
+import com.fcaputo.parkingapp.utils.validation.ValidationErrorType
+import com.fcaputo.parkingapp.utils.validation.ValidationResult
+import java.time.LocalDateTime
+
+class ReservationModel : ReservationContract.Model {
+    private val parkingLotSize = 200 //TODO use the settings value
```

Comment:
as you already have the settings value, you should use it

Reply:
But at this point I don't have communication between both activities, and the size value is in an instance variable, how could I get it in the first place?

Reply:
The parking size could be a singleton! Thus setting it in the `SettingsActivity` and retrieving it in the `ReservationActivity`
https://www.baeldung.com/kotlin/singleton-classes